src/ralph/conf.py

```diff
@@ -80,34 +80,41 @@ def get_instance(self, **init_parameters):
 # Active database backend Settings.
 
 
-class ClickhouseDatabaseBackendSettings(InstantiableSettingsItem):
-    """Pydantic model for ClickHouse database backend configuration settings."""
-
-    _class_path: str = "ralph.backends.database.clickhouse.ClickHouseDatabase"
-
-    HOST: str = "localhost"
-    PORT: int = 8123
-    DATABASE: str = "xapi"
-    EVENT_TABLE_NAME: str = "xapi_events_all"
-    USERNAME: str = None
-    PASSWORD: str = None
-    CLIENT_OPTIONS: dict = None
-
-
 class ClientOptions(BaseModel):
     """Pydantic model for additionnal client options."""
 
     class Config:  # pylint: disable=missing-class-docstring # noqa: D106
         extra = Extra.forbid
 
 
+class ClickhouseClientOptions(BaseModel):
+    """Pydantic model for `clickhouse` client options."""
+
+    class Config:  # pylint: disable=missing-class-docstring # noqa: D106
+        extra = Extra.allow
+
```

Comment:
We could add the parameters used as default in the backend implementation:
```suggestion
class ClickhouseClientOptions(ClientOptions):
    """Pydantic model for `clickhouse` client options."""
    
    date_time_input_format: str = "best_effort"
    allow_experimental_object_type: Literal[0,1] = 1
```